### PR TITLE
added css on the show page for individual houses

### PR DESCRIPTION
**Pull request by SakuraMilkTea8**
Please check!

### CODE REVIEW

File: app/views/houses/show.html.erb [hunk 1]
```diff
@@ -1,49 +1,51 @@
...
-        <%= f.input :end_time, as: :date, html5: true %>
-        <%= f.submit class: 'btn my-2 my-sm-0' %>
+      <h2 class="house-title">
+        <%= @house.name %>
+      </h2>
+<div class="row">
+  <div class="col-6 house-pic">
+    <div class="house-pic-card">
+      <%# make sure to keep the if else tag for the picture %>
+      <% if @house.photo.nil? %>
```
Comment (mvbennett) [inline app/views/houses/show.html.erb:11]: sorry, I might've messed up when I originally wrote this, it should be if @house.photo.key.nil? any way you could write that in quickly?

**Review by mvbennett — CHANGES_REQUESTED** [review]
sorry could you make this one little change to show.html.erb line 11? just need to make sure it says if @house.photo.key.nil?

**Review by mvbennett — APPROVED** [review]
awesome

File: app/views/houses/show.html.erb (hunk 1 above)
Comment (SakuraMilkTea8) [reply app/views/houses/show.html.erb]: Done! Please check :)

**Pull request merged**